test/Generator/ParameterGeneratorTest.php

```diff
@@ -550,6 +550,18 @@ public function testAssigningDefaultValueToVariadicParameterThrowsInvalidArgumen
         $parameter->setDefaultValue([]);
     }
 
+    public function testAssigningDefaultValueToNonVariadicParameter(): void
+    {
+        $parameter = new ParameterGenerator();
+
+        $parameter->setName('parameter');
+        $parameter->setType('int');
+        $parameter->setPosition(1);
+        $parameter->setVariadic(false);
+        $this->expectNotToPerformAssertions();
+        $parameter->setDefaultValue([]);
```

Comment:
This test should verify the generated code

Reply:
There is nothing to verify since there is no assertion for not throwing exceptions.

Reply:
No, I'd expect to see what the generated code looks like for both tests.

Reply:
This is not enough [f3bca9a](https://github.com/laminas/laminas-code/pull/173/commits/f3bca9a1ae6b1c4367ab614e88d622e65313aebe) ?

Reply:
Nay, please do expand the test with the generated code expectation, and remove the `expectNotToPerformAssertions` call.

Reply:
Given this https://github.com/sebastianbergmann/phpunit/issues/2484, I have not idea what we can test here.

Reply:
Or do you think sth like this will work:

```
public function testAssertionPassesWhenFooBarIsAvailable()
{
    try {
        $this->fooBarAvailabilityChecker->assertFooBarAvailable();

        $this->addToAssertionCount(1);
    } catch (FooBarNotAvailable $exception) {
        $this->fail($exception->getMessage());
    }
}
```

As suggested in the thread

Reply:
`self::assertSame('the generated string', $parameter->generate())`

Reply:
Ah, thanks I had no idea  :)